### PR TITLE
Change the customn menu entries

### PR DESCRIPTION
**Pull request by woipi90**
1) if no link is entered u shouldnt be redirected...

2) the submenu image on the right side isnt shown..

http://phpbbsocialnetwork.com/viewtopic.php?f=6&t=4270

### CODE REVIEW

**Comment by Kamahl19** [comment]
you can delete it and push another commit to this PR

File: root/styles/prosilver/template/socialnet/block_menu_item.html [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <!-- BEGIN sn_user_menu -->
-	<li id="menu_id_{sn_user_menu.ID}" class="ui-menu-item">
-		<a href="{sn_user_menu.URL}"<!-- IF sn_user_menu.EXTERNAL --> onclick="window.open(this.href,'_blank');return false;"<!-- ENDIF -->>
+	<li id="menu_id_{sn_user_menu.ID}" class="ui-menu-item" role="presentation">
+		<a href="<!-- IF sn_user_menu.URL neq '' -->{sn_user_menu.URL}<!-- ELSE -->#<!-- ENDIF -->"
+			<!-- IF sn_user_menu.EXTERNAL --> onclick="window.open(this.href,'_blank');return false;"<!-- ENDIF -->>
```
Comment (senky) [inline root/styles/prosilver/template/socialnet/block_menu_item.html:4]: This line should be part of the line above. You cannot divide one html tag into 2 lines according to coding guidelines.

File: root/styles/prosilver/template/socialnet/block_menu_item.html [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 			<!-- IF sn_user_menu.ICON --><span class="sn-menu-icon ui-icon ui-icon-{sn_user_menu.ICON}"></span><!-- ENDIF -->
 		{sn_user_menu.NAME}</a>
 		<!-- IF sn_user_menu.SUBMENU neq '' -->
-		<ul id="submenu_id_{sn_user_menu.ID}" class="ui-menu ui-widget ui-widget-content" style="display:none">{sn_user_menu.SUBMENU}</ul>
```
Comment (senky) [inline root/styles/prosilver/template/socialnet/block_menu_item.html:7]: Why have you deleted this classes?

**Comment by senky** [comment]
Please reword your commit so that it starts with `[i/95]`. Otherwise we cannot merge it. Thanks!

**Pull request closed without merging**